<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModrinthModInfoWidget</class>
 <widget class="QWidget" name="ModrinthModInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="modName">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="modIcon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="TagsFlowWidget" name="tagsWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="modSummary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="WebTextBrowser" name="modDescription">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionCopy_Website_Link">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>Copy Website Link</string>
   </property>
  </action>
  <action name="actionOpen_Website_Link">
   <property name="text">
    <string>Open Website Link</string>
   </property>
  </action>
  <action name="actionOpen_Modrinth_Mod_Dialog">
   <property name="icon">
    <iconset resource="../../images/image.qrc">
     <normaloff>:/image/modrinth.svg</normaloff>:/image/modrinth.svg</iconset>
   </property>
   <property name="text">
    <string>Open Modrinth Mod Dialog</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagsFlowWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/tagsflowwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WebTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header location="global">ui/webtextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/image.qrc"/>
 </resources>
 <connections/>
</ui>
